src/coreclr/vm/stubgen.cpp

```diff
@@ -2355,7 +2400,7 @@ ILStubLinker::ILStubLinker(Module* pStubSigModule, const Signature &signature, S
         ULONG   uStubCallingConvInfo;
         IfFailThrow(m_managedSigPtr.GetCallingConvInfo(&uStubCallingConvInfo));
 
-        if (fStubHasThis)
+        if ((flags & ILSTUB_LINKER_FLAG_STUB_HAS_THIS) != 0)
```

Comment:
How about: `m_fHasThis = (flags & ILSTUB_LINKER_FLAG_STUB_HAS_THIS) != 0;`?

I prefer direction assignment of booleans when possible - up to you.